Core/include/Acts/Seeding2/DoubletSeedFinder.hpp

```diff
@@ -135,8 +156,12 @@ class DoubletsForMiddleSp {
 
  private:
   std::vector<SpacePointIndex2> m_spacePoints;
-  /// contains parameters required to calculate a circle with linear equation
-  std::vector<LinCircle> m_linCircles;
```

Comment:
Assuming `LinCircle` is still used elsewhere?

Reply:
Right the original version of the seeding code still uses this struct but the new one should be decoupled now